web/src/pages/access/request/[id].tsx

```diff
@@ -141,8 +143,25 @@ const Home = () => {
     if (data.when === "scheduled") {
       r.timing.startTime = new Date(data.startDateTime).toISOString();
     }
-    await userCreateRequest(r);
-    navigate({ to: "/requests" });
+    await userCreateRequest(r)
```

Comment:
Will this response need to handle more detail if there are validation errors like user not existing?

Reply:
Done, but problem I'm having is how best to display these errors in the frontend. If it is apart of the error response we return a generic 500 error here and all the helpful meaning is lost. Might need some guidance here @JoshuaWilkes